### PR TITLE
Upload button

### LINKED ISSUE #1: Template designing
We need various templates for the website.
We will refer to this issue.

### PR DESCRIPTION
**Pull request by xeon-zolt**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
it fix the looks of upload button 

#### Changes proposed in this pull request:

- removed links of bootstrap files and added them as files
- updated to bootstrap v4

Fixes #1 and it should be closed after this 
screenshot from 2017-06-20 21-22-25

### CODE REVIEW

File: static/bootstrap.min.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
...onfig.offset,e=this._getScrollHeight(),n=this._config.offset+e-this._getOffsetHeight();if(this._scrollHeight!==e&&this.refresh(),t>=n){var i=this._targets[this._targets.length-1];return void(this._activeTarget!==i&&this._activate(i))}if(this._activeTarget&&t<this._offsets[0]&&this._offsets[0]>0)return this._activeTarget=null,void this._clear();for(var o=this._offsets.length;o--;){var r=this._activeTarget!==this._targets[o]&&t>=this._offsets[o]&&(void 0===this._offsets[o+1]||t<this._offsets[o+1]);r&&this._activate(this._targets[o])}},h.prototype._activate=function(e){this._activeTarget=e,this._clear();var n=this._selector.split(",");n=n.map(function(t){return t+'[data-target="'+e+'"],'+(t+'[href="'+e+'"]')});var i=t(n.join(","));i.hasClass(_.DROPDOWN_ITEM)?(i.closest(g.DROPDOWN).find(g.DROPDOWN_TOGGLE).addClass(_.ACTIVE),i.addClass(_.ACTIVE)):i.parents(g.LI).find("> "+g.NAV_LINKS).addClass(_.ACTIVE),t(this._scrollElement).trigger(f.ACTIVATE,{relatedTarget:e})},h.prototype._clear=function(){t(this._selector).filter(g.ACTIVE).removeClass(_.ACTIVE)},h._jQueryInterface=function(e){return this.each(function(){var n=t(this).data(a),o="object"===("undefined"==typeof e?"undefined":i(e))&&e;
```
Comment (xeon-zolt) [inline static/bootstrap.min.js]: @houndci-bot that is official bootstrap js :-1:

**Comment by meets2tarun** [comment]
@xeon-zolt Please provide a test-URL of your made changes, so that it would be easy for the one to review.
Thanks

File: static/bootstrap.min.js (hunk 1 above)
Comment (abishekvashok) [inline static/bootstrap.min.js]: @xeon-zolt that's a bot i recon

**Comment by xeon-zolt** [comment]
test url --> https://meilix-generator-test.herokuapp.com/

File: templates/index.html [hunk 2]
```diff
@@ -29,13 +29,12 @@ <h2 align="center">Meilix Generator</h2>
 <div class="col-md-06">
     <div class="form-group">
         <label>Upload Wallpaper:</label>
-        <div class="input-group">
-            <span class="input-group-btn">
-                <span class="btn btn-default btn-file">
-                    <input type="file" name="file" class="file" data-show-preview="false" required>
```
Comment (meets2tarun) [inline templates/index.html]: Which wallpaper will we set if the user doesn't upload any wallpaper?

**Comment by meets2tarun** [comment]
os
There is difference in the screenshot uploaded and in this, in the rectangle surrounded the `Browse` button and the email, event name and upload wallpaper boldness.
There is very less upper padding, this should be increased a bit (better the earlier padding)

**Comment by xeon-zolt** [comment]
the changes happened after i updated bootstrap

File: templates/index.html (hunk 2 above)
Comment (xeon-zolt) [inline templates/index.html]: dude why are you commenting in history ?

**Comment by xeon-zolt** [comment]
screenshot from 2017-06-22 00-53-01

File: templates/index.html (hunk 2 above)
Comment (meets2tarun) [inline templates/index.html]: So better tell me the way to review the changes which I want in this PR.

**Comment by abishekvashok** [comment]
OK pls squash your commits. Let's get professional :) @xeon-zolt

**Comment by meets2tarun** [comment]
Top Padding needs to be increased (better the older one) and `required` option need to be there on the Upload button.

File: templates/index.html (hunk 2 above)
Comment (xeon-zolt) [inline templates/index.html]: it confuses i would like comment on my code

**Comment by xeon-zolt** [comment]
@meets2tarun by top padding you mean padding of ? button ? or whole form?

**Comment by meets2tarun** [comment]
Just one thing before it gets merge;
If a user doesn't upload a wallpaper, the build button will not proceed. Working
But it should give a pop-up to the user to upload the wallpaper; need to be fixed.

**Pull request merged**